doc: update technical escalation proccess (#63587)
zephyrproject-rtos/zephyr: doc/project/project_roles.rst

name to one or more ``collaborators`` sections of the :ref:`maintainers_file` in
the Zephyr repository.

Collaborator change requests on pull requests should
be addressed by the original submitter. In cases where the changes requested do
not follow the :ref:`expectations <reviewer-expectations>` and the guidelines
of the project or in cases of disagreement, it is the responsibility of the
assignee to advance the review process and resolve any disagreements.

> Approval of pull requests by collaborator is counted toward the minimum

Reviewer: ```suggestion
Collaborator approval of pull requests are counted toward the minimum

```

name to one or more ``collaborators`` sections of the :ref:`maintainers_file` in
the Zephyr repository.

Collaborator change requests on pull requests should
be addressed by the original submitter. In cases where the changes requested do
not follow the :ref:`expectations <reviewer-expectations>` and the guidelines
of the project or in cases of disagreement, it is the responsibility of the
assignee to advance the review process and resolve any disagreements.

Collaborator approval of pull requests are counted toward the minimum

> 

Reviewer: Extra line.

GitHub user name to one or more ``maintainers`` sections of the
:ref:`maintainers_file` in the Zephyr repository.

> Approval of pull requests by maintainers is counted toward the minimum

Reviewer: ```suggestion
Maintainer approval of pull requests are counted toward the minimum
```
